plenum/common/messages/fields.py

```diff
@@ -670,7 +670,10 @@ class ProtocolVersionField(FieldBase):
     _base_types = (int, type(None))
 
     def _specific_validation(self, val):
-        if val is None:
-            return
         if not PlenumProtocolVersion.has_value(val):
-            return 'Unknown protocol version value {}'.format(val)
+            return 'Unknown protocol version value {}. ' \
+                   'Update your client application.'.format(val)
```

Comment:
It's better to say `please update libindy or indy-client to the latest stable version`

Reply:
ok

Reply:
ok